### PR TITLE
Ba meyer master

### PR DESCRIPTION
**Pull request by lqwert**
Merge so Front-Endbackend are in sync

### CODE REVIEW

File: src/app/core/component/rating/rating.component.ts [hunk 1]
```diff
@@ -9,21 +12,99 @@ import { Rating } from '../../model/rating.enum';
...
+      const indexDown = this.downVotes.indexOf(ratingModel.userId);
+      if (indexDown > -1) this.downVotes.splice(indexDown, 1);
+    }
   }
 
-  down(){
-    this.userRatingCurrent.emit(Rating.DOWN);
-  } 
+  click(rating: number) {
+    switch (this.context) {
```
Comment (manuwei) [inline src/app/core/component/rating/rating.component.ts:66]: what do the number of context mean exactly? could you use an enum instead?

File: src/app/core/issue-management/_store/issue-management-store.ts [hunk 2]
```diff
@@ -1,18 +1,35 @@
...
+import { Injectable } from '@angular/core';
 
+@Injectable()
 export class IssueManagementStore {
-    private _issue2Candidate: BehaviorSubject<any> = new BehaviorSubject(null);
 
-    get candidateFromIssue() {
-      return this._issue2Candidate.asObservable();
+    private _issue: BehaviorSubject<any> = new BehaviorSubject(null);
+    // private _issue2Candidate: BehaviorSubject<any> = new BehaviorSubject(null);
```
Comment (manuwei) [inline src/app/core/issue-management/_store/issue-management-store.ts:9]: //?

File: src/app/core/issue-management/_store/issue-management-store.ts [hunk 3]
```diff
@@ -1,18 +1,35 @@
...
+      this._issue.next(issue);
     }
 
-    resetCandidateFromIssue() {
-      this._issue2Candidate.next(null);
+    resetIssue() {
+      this._issue.next(null);
     }
+
+  // addIssue2Candidate(issue2Candidate: Issue) {
```
Comment (manuwei) [inline src/app/core/issue-management/_store/issue-management-store.ts:27]: //??

File: src/styles.scss [hunk 4]
```diff
@@ -241,3 +245,72 @@ body {
...
+  padding: 4px 0;
+}
+
+.buttons {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+}
+
+// .scrollable-double {
```
Comment (manuwei) [inline src/styles.scss:301]: //?

File: src/app/core/component/rating/rating.component.ts (hunk 1 above)
Comment (lqwert) [reply src/app/core/component/rating/rating.component.ts:66]: Added an enum [Context](https://github.com/PatternAtlas/pattern-atlas-ui/blob/4c29ad8d9d9cb333488ba9c066ff8f33717d9893/src/app/core/shared/_enums/context.enum.ts)

File: src/app/app-routing.module.ts [hunk 5]
```diff
@@ -77,10 +79,15 @@ const routes: Routes = [
     path: 'graph',
     component: PatternLanguageGraphComponent
   },
+  {
+    path: '',
```
Comment (marzn) [inline src/app/app-routing.module.ts:83]: This is already defined in L29

**Comment by manuwei** [comment]
we should integrate the auth-server into the dockersetup before we merge this branch - in the local setup the database and the auth-serve are provided on the same port?

**Comment by lqwert** [comment]
> we should integrate the auth-server into the dockersetup before we merge this branch - in the local setup the database and the auth-serve are provided on the same port?

No I am using the same database and port as the PatternPediaApi, so the setup is:
PatternPediaApi: Port 8080
PatternPediaAuth: Port 8081
DB: Port 5432

So right now PatternPediaApi & PatternPediaAuth are sharing the user table. Is this ok or should i change it so the api and auth server having two seperate dbs?

**Pull request closed without merging**